pandas/tests/frame/test_dtypes.py

```diff
@@ -850,6 +850,24 @@ def test_arg_for_errors_in_astype(self):
 
         df.astype(np.int8, errors='ignore')
 
+    def test_arg_for_errors_in_astype_dictlist(self):
+        data_df = pd.DataFrame([{'col_a': '1', 'col_b': '16.5%',
+                                 'col_c': 'test'},
+                                {'col_a': '2.2', 'col_b': '15.3',
```

Comment:
this can be formatted a bit nicer by
```
pd.DataFrame(
           {
           ....
          })
```

also rather than using 'col_a', just call this 'a'

Reply:
This is the example code from issue #25905  as requested...that will still be able to be closed if it is modified in this manner?

Reply:
Formatted in 1c35cc70108a0407d964c26b5cfc07d654dd4ebc